frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/redux/stateUpdate/vmSettings/prefill-vm-template-state-update.ts

```diff
@@ -177,18 +180,31 @@ export const prefillVmTemplateUpdater = ({ id, dispatch, getState }: UpdateOptio
         }
       }
 
+      let type = diskWrapper.isFirstBootableDevice()
+        ? VMWizardStorageType.PROVISION_SOURCE_TEMPLATE_DISK
+        : VMWizardStorageType.TEMPLATE;
+
+      if (
+        Object.values(WINTOOLS_CONTAINER_NAMES).find(
```

Comment:
Can we make a function out of this? `isWinToolsImage` ?

Reply:
fixed @suomiy